documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/websocket/server-websocket.adoc

```diff
@@ -12,6 +12,33 @@
 //
 
 [[pg-server-websocket]]
-=== WebSocket Server Libraries
+=== WebSocket Server
 
-TODO
+Jetty provides two API implementations of the WebSocket protocol:
+
+* An implementation for the standard `javax.websocket` APIs provided by link:https://www.jcp.org/en/jsr/detail?id=356[JSR 356], described in xref:pg-server-websocket-standard[this section].
+* An implementation for Jetty-specific WebSocket APIs, described in xref:pg-server-websocket-jetty[this section].
+
+Using the standard `javax.websocket` APIs allows your applications to depend only on standard APIs, and your applications may be deployed in any compliant WebSocket Container that supports JSR 356.
+
+The standard APIs provide few features that are not present in the Jetty WebSocket APIs:
+
+* Encoders and Decoders for automatic conversion of text or binary messages to objects.
+* `Reader` and `InputStream` for simple, blocking, message streaming.
+* Simple URI template matching.
+
+On the other hand, the Jetty WebSocket APIs are more efficient and offer greater and more fine-grained control, and provide features that are not present in the standard APIs:
+
+* `MessageSink` for advanced, asynchronous, message streaming with backpressure.
```

Comment:
`MessageSink` is a websocket-core internal interface. It's not one of the features of the Jetty WebSocket API.

Reply:
Removed.